.gitignore

```diff
@@ -1,4 +1,4 @@
 .bundle
 bin/configlet
 bin/configlet.exe
-Gemfile.lock
+/Gemfile.lock
```

Comment:
Which "This"?  The one that I propose does the same, but specifically also says to NOT ignore the files in each of the exercise subfolders (but only the root subfolder.), while also ignoring Gemfile.lock anywhere else.